### PR TITLE
Mobile Release v1.67.0

### PR DESCRIPTION
**Pull request by SiobhyB**
## Description
Release 1.67.0 of the react-native-editor and Gutenberg-Mobile.

For more information about this release and testing instructions, please see the related Gutenberg-Mobile PR: https://github.com/wordpress-mobile/gutenberg-mobile/pull/4289

## Checklist:
- [ ] My code is tested.
- [ ] My code follows the WordPress code style. 
- [ ] My code follows the accessibility standards. 
- [ ] I've tested my changes with keyboard and screen readers. 
- [ ] My code has proper inline documentation. 
- [ ] I've included developer documentation if appropriate. 
- [ ] I've updated all React Native files affected by any refactorings/renamings in this PR (please manually search all `*.native.js` files for terms that need renaming or removal).

### CODE REVIEW

File: packages/react-native-editor/CHANGELOG.md [hunk 1]
```diff
@@ -10,6 +10,8 @@ For each user feature we should also add a importance categorization label  to i
 -->
 
 ## Unreleased
+
+## 1.67.0
 -   [*] [Embed block] Included Link in Block Settings [#36099]
 -   [**] Fix tab titles translation of inserter menu [#36534]
 -   [*] [Media&Text block] Fix an issue where the text font size would be bigger than expected in some cases [#36570]
```
Comment (jd-alexander) [inline packages/react-native-editor/CHANGELOG.md]: @SiobhyB what do you think about updating this changelog entry to add the spaces? LMK 🙇🏾 
```suggestion
-   [*] [Media & Text block] Fix an issue where the text font size would be bigger than expected in some cases [#36570]
```

Comment (SiobhyB) [reply packages/react-native-editor/CHANGELOG.md]: Sounds good @jd-alexander! Updated in https://github.com/WordPress/gutenberg/pull/36848/commits/93a041a4173e7747267cb8279451753df263c88b and also added a missing change in https://github.com/WordPress/gutenberg/pull/36848/commits/88cb6a632b8d4a1505a25c63637358f6de4a7916. I'll go through to update the connected PRs too.

**Review by antonis — APPROVED** [review]
Thank you for wranging this release @SiobhyB 🙇 
LGTM 🎉

**Review by fluiddot — APPROVED** [review]
LGTM 🎊 !

**Pull request merged**